src/main.ts

```diff
@@ -18,23 +19,40 @@ export async function run() {
 
   try {
     // Automatically configure bridge if Bridge download url is provided
-    if (inputs.BRIDGE_DOWNLOAD_URL) {
-      // Download file in temporary directory
-      info('Downloading and configuring Synopsys Bridge')
-      const downloadResponse: DownloadFileResponse = await getRemoteFile(tempDir, inputs.BRIDGE_DOWNLOAD_URL)
-      const extractZippedFilePath: string = inputs.SYNOPSYS_BRIDGE_PATH || getBridgeDefaultPath()
-
-      // Clear the existing bridge, if available
-      if (fs.existsSync(extractZippedFilePath)) {
-        const files: string[] = fs.readdirSync(extractZippedFilePath)
-        for (const file of files) {
-          await rmRF(file)
-        }
+    let bridgeUrl = ''
+
+    const sb = new SynopsysBridge()
+    if (inputs.BRIDGE_DOWNLOAD_VERSION) {
+      if (await sb.validateBridgeVersion(inputs.BRIDGE_DOWNLOAD_VERSION)) {
+        bridgeUrl = sb.getVersionUrl(inputs.BRIDGE_DOWNLOAD_VERSION.trim()).trim()
+      } else {
+        return Promise.reject(new Error('The provided bridge version '.concat(inputs.BRIDGE_DOWNLOAD_VERSION).concat(' is not available in artifactory')))
```

Comment:
Message can be-" Provided bridge version <version> not found in antifactory"

Reply:
Done.